public/app/features/alerting/unified/hooks/ruleGroup/useDeleteRuleFromGroup.ts

```diff
@@ -20,14 +21,12 @@ export function useDeleteRuleFromGroup() {
   const [upsertRuleGroup] = alertRuleApi.endpoints.upsertRuleGroupForNamespace.useMutation();
   const [deleteRuleGroup] = alertRuleApi.endpoints.deleteRuleGroupFromNamespace.useMutation();
 
-  return useAsync(async (ruleGroup: RuleGroupIdentifier, rule: RulerRuleDTO) => {
+  return useAsync(async (ruleGroup: RuleGroupIdentifier, ruleIdentifier: EditableRuleIdentifier) => {
```

Comment:
This is very unfortunate, but `RuleIdentifier` contains overlapping information with `RuleGroupIdentifier` but _only_ for cloud (data source managed) rules and not for Grafana-managed rules, so we still need both function arguments 🫠